services/QuillLMS/README.md

```diff
@@ -110,6 +110,9 @@ bundle exec rspec spec
 
 - frontend
   - ESLint: from the root directory run `npm run eslint`
+  - Tests: `npm run jest`
+   - For an individual test you can run: `npm run jest [text]` which will just wildcard whatever the text is -- so if I want to run all the tests in Connect I just run npm run jest Connect.
+   - `npm run jest [your/file/path]`
```

Comment:
Thank you for adding this.  For more context, you can add a link to the package.json [instruction](https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/package.json#L20).